export default [
  {
    id: 1,
    value: "transcript_variant",
    label: "Transcript variant",
    children: [2, 19, 25, 29, 30],
  },
  {
    id: 2,
    value: "coding_variant",
    label: "Coding variant",
    children: [3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18],
    parent: 1,
  },
  {
    id: 3,
    value: "coding_sequence_variant",
    label: "Coding sequence variant",
    n: 18057,
    description: "A sequence variant that changes the coding sequence",
    parent: 2,
  },
  {
    id: 4,
    value: "feature_elongation",
    label: "Feature elongation",
    n: 0,
    description:
      "A sequence variant that causes the extension of a genomic feature, with regard to the reference sequence",
    parent: 2,
  },
  {
    id: 5,
    value: "feature_truncation",
    label: "Feature truncation",
    n: 0,
    description:
      "A sequence variant that causes the reduction of a genomic feature, with regard to the reference sequence",
    parent: 2,
  },
  {
    id: 6,
    value: "frameshift_variant",
    label: "Frameshift variant",
    n: 506417,
    description:
      "A sequence variant which causes a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three",
    parent: 2,
  },
  {
    id: 7,
    value: "incomplete_terminal_codon_variant",
    label: "Incomplete terminal codon variant",
    n: 3443,
    description:
      "A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed",
    parent: 2,
  },
  {
    id: 8,
    value: "inframe_deletion",
    label: "Inframe deletion",
    n: 143302,
    description:
      "An inframe non synonymous variant that deletes bases from the coding sequence",
    parent: 2,
  },
  {
    id: 9,
    value: "inframe_insertion",
    label: "Inframe insertion",
    n: 50924,
    description:
      "An inframe non synonymous variant that inserts bases into in the coding sequence",
    parent: 2,
  },
  {
    id: 10,
    value: "missense_variant",
    label: "Missense variant",
    n: 12385885,
    description:
      "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved",
    parent: 2,
  },
  {
    id: 11,
    value: "NMD_transcript_variant",
    label: "NMD transcript variant",
    n: 20962310,
    description: "A variant in a transcript that is the target of NMD",
    parent: 2,
  },
  {
    id: 12,
    value: "protein_altering_variant",
    label: "Protein altering variant",
    n: 3482,
    description:
      "A sequence_variant which is predicted to change the protein encoded in the coding sequence",
    parent: 2,
  },
  {
    id: 13,
    value: "synonymous_variant",
    label: "Synonymous variant",
    n: 6340518,
    description:
      "A sequence variant where there is no resulting change to the encoded amino acid",
    parent: 2,
  },
  {
    id: 14,
    value: "start_lost",
    label: "Start lost",
    n: 38719,
    description:
      "A codon variant that changes at least one base of the canonical start codon",
    parent: 2,
  },
  {
    id: 15,
    value: "stop_gained",
    label: "Stop gained",
    n: 402831,
    description:
      "A sequence variant whereby at least one base of a codon is changed, resulting in a premature stop codon, leading to a shortened transcript",
    parent: 2,
  },
  {
    id: 16,
    value: "stop_lost",
    label: "Stop lost",
    n: 16247,
    description:
      "A sequence variant where at least one base of the terminator codon (stop) is changed, resulting in an elongated transcript",
    parent: 2,
  },
  {
    id: 17,
    value: "start_retained_variant",
    label: "Start retained variant",
    n: 585,
    description:
      "A sequence variant where at least one base in the start codon is changed, but the start remains",
    parent: 2,
  },
  {
    id: 18,
    value: "stop_retained_variant",
    label: "Stop retained variant",
    n: 7754,
    description:
      "A sequence variant where at least one base in the terminator codon is changed, but the terminator remains",
    parent: 2,
  },
  {
    id: 19,
    label: "Non-coding variant",
    children: [20, 21, 22, 23, 24],
    parent: 1,
  },
  {
    id: 20,
    value: "3_prime_UTR_variant",
    label: "3 prime UTR variant",
    n: 4109311,
    description: "A UTR variant of the 3' UTR",
    parent: 19,
  },
  {
    id: 21,
    value: "5_prime_UTR_variant",
    label: "5 prime UTR variant",
    description: "A UTR variant of the 5' UTR",
    parent: 19,
    children: [43],
  },
  {
    id: 22,
    value: "intron_variant",
    label: "Intron variant",
    n: 212366571,
    description: "A transcript variant occurring within an intron",
    parent: 19,
  },
  {
    id: 23,
    value: "non_coding_transcript_exon_variant",
    label: "Non coding transcript exon variant",
    n: 7450628,
    description:
      "A sequence variant that changes non-coding exon sequence in a non-coding transcript",
    parent: 19,
  },
  {
    id: 24,
    value: "non_coding_transcript_variant",
    label: "Non coding transcript variant",
    n: 52390424,
    description: "A transcript variant of a non coding RNA gene",
    parent: 19,
  },
  {
    id: 25,
    label: "Splice variant",
    children: [26, 27, 28],
    parent: 1,
  },
  {
    id: 26,
    value: "splice_acceptor_variant",
    label: "Splice acceptor variant",
    n: 172987,
    description:
      "A splice variant that changes the 2 base region at the 3' end of an intron",
    parent: 25,
  },
  {
    id: 27,
    value: "splice_donor_variant",
    label: "Splice donor variant",
    n: 210311,
    description:
      "A splice variant that changes the 2 base region at the 5' end of an intron",
    parent: 25,
  },
  {
    id: 28,
    value: "splice_region_variant",
    label: "Splice region variant",
    n: 2785532,
    description:
      "A sequence variant in which a change has occurred within the region of the splice site, either within 1-3 bases of the exon or 3-8 bases of the intron",
    parent: 25,
  },
  {
    id: 29,
    value: "transcript_ablation",
    label: "Transcript ablation",
    n: 18,
    description:
      "A feature ablation whereby the deleted region includes a transcript feature",
    parent: 1,
  },
  {
    id: 30,
    value: "transcript_amplification",
    label: "Transcript amplification",
    n: 0,
    description: "A feature amplification of a region containing a transcript",
    parent: 1,
  },
  {
    id: 31,
    label: "Regulatory variant",
    children: [32, 33, 34, 35, 36, 37, 38],
  },
  {
    id: 32,
    value: "mature_miRNA_variant",
    label: "Mature miRNA variant",
    n: 6871,
    description:
      "A transcript variant located with the sequence of the mature miRNA",
    parent: 31,
  },
  {
    id: 33,
    value: "regulatory_region_ablation",
    label: "Regulatory region ablation",
    n: 0,
    description:
      "A feature ablation whereby the deleted region includes a regulatory region",
    parent: 31,
  },
  {
    id: 34,
    value: "regulatory_region_amplification",
    label: "Regulatory region amplification",
    n: 0,
    description:
      "A feature amplification of a region containing a regulatory region",
    parent: 31,
  },
  {
    id: 35,
    value: "regulatory_region_variant",
    label: "Regulatory region variant",
    n: 0,
    description: "A sequence variant located within a regulatory region",
    parent: 31,
  },
  {
    id: 36,
    value: "TF_binding_site_variant",
    label: "TF binding site variant",
    n: 0,
    description:
      "A sequence variant located within a transcription factor binding site",
    parent: 31,
  },
  {
    id: 37,
    value: "TFBS_ablation",
    label: "TFBS ablation",
    n: 0,
    description:
      "A feature ablation whereby the deleted region includes a transcription factor binding site",
    parent: 31,
  },
  {
    id: 38,
    value: "TFBS_amplification",
    label: "TFBS amplification",
    n: 0,
    description:
      "A feature amplification of a region containing a transcription factor binding site",
    parent: 31,
  },
  {
    id: 39,
    label: "Intergenic variant",
    children: [40, 41, 42],
  },
  {
    id: 40,
    value: "downstream_gene_variant",
    label: "Downstream gene variant",
    n: 0,
    description: "A sequence variant located 3' of a gene",
    parent: 39,
  },
  {
    id: 41,
    value: "intergenic_variant",
    label: "Intergenic variant",
    n: 38329053,
    description:
      "A sequence variant located in the intergenic region, between genes",
    parent: 39,
  },
  {
    id: 42,
    value: "upstream_gene_variant",
    label: "Upstream gene variant",
    n: 0,
    description: "A sequence variant located 5' of a gene",
    parent: 39,
  },
  {
    id: 43,
    value: "shine_dalgarno_sequence",
    label: "Shineâ€“Dalgarno sequence",
    description:
      "ribosomal binding site in bacterial and archaeal messenger RNA",
    parent: 21,
    children: [44],
  },
  {
    id: 44,
    value: "pyrimidine",
    label: "Pyrimidine",
    description: "aromatic heterocyclic organic compound similar to pyridine",
    parent: 43,
    children: [45],
  },
  {
    id: 45,
    value: "pyrimidine123",
    label: "Pyrimidine123",
    n: 1546374,
    description: "aromatic heterocyclic organic compound similar to pyridine",
    parent: 44,
  },
];
